geoips/commandline/commandline_interface.py

```diff
@@ -76,6 +76,13 @@ def __init__(self, instructions_dir=None, legacy=False):
     def execute_command(self):
         """Execute the given command."""
         self.GEOIPS_ARGS = self.parser.parse_args()
+        # NOTE: We should discuss how we want to share the selected LOG to child classes
+        # They can all use the functionality below, however that would be redundant and
+        # there is likely a better way to fix this. Unfortunately 'parse_known_args'
+        # didn't work and this is our best solution for the time being.
+
+        # self.LOG = getattr(LOG, self.GEOIPS_ARGS.log_level)
+        # self.LOG("LOG LEVEL = {self.GEOIPS_ARGS.log_level}")
```

Comment:
Is there an issue for this? If not, we should create it.

Reply:
I'll create that issue.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/714